using-babelfish/getting-started.md

```diff
@@ -61,7 +61,7 @@ Exposing the port \(`9432`\) with `-p 9432:9432` makes it easier to connect to t
 If you are behind an HTTP or HTTPS proxy server, for example in corporate settings, you will need to add the `HTTP_PROXY`, `HTTPS_PROXY`, and `NO_PROXY` environment variables in the docker run command to configure HTTP or HTTPS proxy behavior.
 
 ```bash
-$ docker run -d --name bblfshd --privileged -p 9432:9432 -e 
+$ docker run -d --name bblfshd --privileged -p 9432:9432 -e
 HTTP_PROXY="http://proxy.example.com:80/" -v bblfshd-cache:/var/lib/bblfshd bblfsh/bblfshd
```

Comment:
While we are here, can you please remove a new line after `-e`?

Reply:
May be it's also ok to remove this later?